apps/foundation-demo/src/app/misc/header/actions/sidebar.actions.ts

```diff
@@ -0,0 +1,14 @@
+import { Action } from '@ngrx/store';
+
+export enum SidebarActionTypes {
+  ToggleShowSidebarAction = "[Foundation-Sidebar] Toggle Show Sidebar Action"
```

Comment:
Can we call this "ToggleSidebarVisibilityAction"? I think the naming convention is a bit cleaner.